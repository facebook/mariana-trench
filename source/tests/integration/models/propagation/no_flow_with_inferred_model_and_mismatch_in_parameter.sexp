(method (private) "LFlow;.into_sink:(LData;LData;)V"
 (
  (load-param v0)
  (load-param v4)
  (load-param v5)

  LINE(1)
  (invoke-virtual (v0 v5) "LSink;.sink:(LData;)V")
  (return-void)
 )
)
(method (private) "LFlow;.indirect:()V"
 (
  LINE(1)
  (invoke-virtual (v0) "LSource;.source:()LData;")
  (move-result-object v2)

  LINE(2)
  (invoke-virtual (v0 v2 v1)
  "LFlow;.into_sink:(LData;LData;)V") (return-void)
 )
)
