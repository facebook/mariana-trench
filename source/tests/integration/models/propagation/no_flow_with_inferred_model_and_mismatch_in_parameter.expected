{
  "metadata" :
  {
    "@generated" : true
  },
  "models" :
  [
    {
      "method" : "LFlow;.indirect:()V",
      "position" :
      {
        "line" : 0,
        "path" : "Flow.java"
      }
    },
    {
      "method" : "LFlow;.into_sink:(LData;LData;)V",
      "position" :
      {
        "line" : 0,
        "path" : "Flow.java"
      },
      "sinks" :
      [
        {
          "call_position" :
          {
            "line" : 1,
            "path" : "Flow.java"
          },
          "callee" : "LSink;.sink:(LData;)V",
          "callee_port" : "Argument(1)",
          "caller_port" : "Argument(2)",
          "distance" : 1,
          "kind" : "TestSink",
          "origins" :
          [
            "LSink;.sink:(LData;)V"
          ]
        }
      ]
    }
  ]
}
